docs/DeveloperGuide.adoc

```diff
@@ -844,16 +901,67 @@ Use case ends.
 +
 Use case resumes at step 2.
 
-_{More to be added}_
+[discrete]
+=== Use case: Sort by locality
+
+*MSS*
+
+1.  User types “/sortl” in to the command line.
+2.  Addressbook will prompt user to select whether they would like to view the list for students or tutors.
+3.  User will enter either “s” for students or “t” for tutors.
+4.  Addressbook will return a list of students or tutors based sorted by their locality.
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 3a. System detects an invalid input.
++
```

Comment:
Should there also be an extension for if the user wants to escape from the command input? (Such as if the user changes his mind after typing `/sortl`

Reply:
Will update in my commit